pyproject.toml

```diff
@@ -17,7 +17,7 @@ classifiers = [
 ]
 
 [project.optional-dependencies]
-testing = ["assess", "pandas"]
+testing = ["assess", "pandas", "shapely"]
```

Comment:
Do we need to add this dependency to gadopt in such a way that those doing --install gadopt after this merge automatically get shapely? This maybe a dumb point, but flagging in case.

Reply:
I agree that having `shapely` under optional testing dependencies does not feel right. Either we make it a required dependency of `g-adopt` or we add an optional `multi-material` dependency alongside `testing` and `optimisation`.

Reply:
Having thought a little bit more about it, I think it is fine the way it is.